### ISSUE TITLE
"Require by" vs. "Required by" depending on whether or not a type is required

### ISSUE THREAD

**Issue by menewman** [GraphiDocsOrg/docs#27]
This shows up for a type that has at least one "required by" element:
![Screen Shot 2019-03-22 at 11 01 44 AM](https://user-images.githubusercontent.com/747378/54850701-12c92e00-4ca5-11e9-8690-5645816bd269.png)

This shows up for a type that has no "required by" elements:
![Screen Shot 2019-03-22 at 10 53 31 AM](https://user-images.githubusercontent.com/747378/54850707-178de200-4ca5-11e9-9cd2-8b3f9ef34ef6.png)

Basically, it seems like the goal of [this commit](https://github.com/GraphiDocsOrg/docs/commit/f7452068f6a25a41dd263e6b7de55e6a60f79221) was to change the wording from "Require By" to "Required By" (which I agree with -- the latter wording is correct!), but it just missed the edge case where there are no required-by items.
